lib/scholar/naive_bayes/gaussian_naive_bayes.ex

```diff
@@ -0,0 +1,442 @@
+defmodule Scholar.NaiveBayes.Gaussian do
+  @moduledoc """
+  Gaussian Naive Bayes.
```

Comment:
Perhaps that Stanford reference could be added as a link in the extended doc, as well as a short explanation on what this does